DEX Integration Docs

oeggert opened the pull request:
How-to guide to connect and trade on the XRPL DEX.

AKCodez committed:
Migrate marketing pages to Redocly

adds events page, updates convert-template script

adds proper filter logic to events page, adds moment

converts history page

converts impact and xrpl-ledger-overview page

try getting animation on impact to work

converts xrp overview page and logic

adds contribute page, still needs typeform integration and animations

converts developer funding page

adds dev tools page

add missing image

adds code samples py conversion to js

adds hook to read current theme, adds animations to impact page

adds careers animations

adds correct animations for contribute page

adds light mode v of animations on contribute page

adds animations to uses page

adds modal logos and uses modal logic

completes uses page

more changes

Fix casing issues with use case files

fix grid issue on uses

RomanHotsiy committed:
chore: basic impl of code-samples page

mDuo13 committed:
Updated redocly sidebar generation

mDuo13 committed:
Markdown syntax migration script

Script replacing include_code with code-snippet

Migration script: partials, some variables

Add variables to conversion script

draft repo-link component

Complete repo-link component

Migration script: handle github links

Draft include_svg→inline-svg (non-functional)

Currently doesn't work due to image path issues.
Also, captions and custom classes (for inlining) not implemented yet.

Conversion script: refactor & add code-page-name

Custom code-page-name component works around Markdoc limitation where
vars can't be used in `inline code` sections.

Migrate script: Handle more code includes correctly

Migration script: tabs and tabbed code samples

Child pages macro & conversion script

Adapted from 70cffa67eddd870c738707facc8552146eef3b9a

Migration script: update with some partial fixes

Migration script: callouts→admonitions

Fix auto-generation of index pages

Migration script: fix SVG migration

Migration scripting: fix code block prefixes & indentation

- Use the Redocly 0.66 feature for code block prefixes
- Update the script for converting indented code blocks to fences with
  Roman's latest fixes (now uses 4 spaces per level, for consistency)

JST5000 committed:
Migrate the tx-sender page to Redocly

Update convert-template

Add basic page

Add it to the sidebar

Fix a broken link

Fix translate usage and add linebreaks

Fix indents

Add basic sidebar

Port over init button logic

Migrate submit_and_notify and start dest addr

Get the payment button working

Componentize the Send XRP Payment button

Add basic escrow button

Componentize payment channel

Migrate Trust For

Migrate Send Issued Currency

Add partial payment progres bar logic

Use the component for the partial payment

Add support for escrow finish

Log transactions in sidebar

Debugging partial payment setup

Add support for changing destinationAddress

Finish adding bootstrap growl notifications

Use 'client' instead of 'api'

Move DestinationAddressInput to component and remove ids

Split the page into separate files

Remove the old files for this page

Update links

Add space

Add comment deprecating bootstrap-growl jquery

Fix typing errors

PR Comments Pt 1

Small PR fixes

Encapsulate isValidDestinationAddress

JST5000 committed:
Fix top-nav padding

mDuo13 committed:
Rename DepositAuth table to md for compatibility

JST5000 committed:
Migrate toml checker tool

Add baseline html

Add script tags

WIP fetchWallet

Make basic page with account step 1 work

Add decodeHex and helpers to update logs

Add fetchFile to access toml from domain

Copy over code & comment out migrated pieces

Add toml parsing

WIP: Add types and uncomment new code

Update the parseToml function to share code

Mostly migrate the validateDomain function

Fix bug by using for instead of foreach

Clean up code part 1

Refactor into separate files

Translate everything

Componentize the buttons

Split out code into separate files

Update package-lock

Fix spacing and uncomment code

Fix indentation

Fix direct import of xrpl

Fix import

cleaned up log entry handling to not build an array of elements

moved to resource folder and update css.

Move shared components and fix small errors

Move file and update sidebars

Fix slow load of long list of addresses

toml checker - sidebar/width fixes

mDuo13 committed:
Fix multicode typo in rhel install

mDuo13 committed:
Fix broken images in some pages

mDuo13 committed:
Bump Redocly to 0.65.6

mDuo13 committed:
Add missing final newlines

mDuo13 committed:
Redocly: bump portal to v0.65.7

mDuo13 committed:
Bump Redocly to v0.66.0

ckniffen committed:
Move font-awesome css to better reflect relative paths

ckniffen committed:
Use `@uiw/react-codemirror` and `react@18`

Updating redocly to latest and leveraging `@uiw/react-codemirror` which
it also uses.

ckniffen committed:
Move styles building up to the project root

This will make development much easier by having all commands at the
project root. This also removes the deprecated `node-sass`

Fix build-css-watch

JST5000 committed:
Migrate the faucet page to Redocly (#2243)

* Get basic HTML loading for faucet page

* Add xrpl.js implementation

* Add sidebar and fix throbber

* Add translates

* Try to format sidebar

* Fix formatting

* Support xrpl.js

* Fix links

* Comment out XRPLGuard for now

* Make AMM Devnet faucet work

* Improve readability

* Update all instances of link + fix topnav

* Remove unnecessary file

* Use a more current version of xrpl

* Add missing loader while keys are generating

* Type with xrpl and remove unnecessary script

* Use string interpolation instead of multiple trans

* Move faucets into a json file

* Remove the old faucet code

* Use xrpl-beta directly

* Use dropsToXRP

* Support hooks natively

* Remove AMM-Devnet

* Revert changes to link path

* Revert link changes pt 2

* Revert pt 3

* Use XRPLoader for loading icon

* Fix small mistakes

* Remove unnecessary changes

mDuo13 committed:
Create script to migrate links and content syntax

Add tool/migrate.sh as a one-stop conversion script for the whole repo.
This script's duties include:

- Changing all links from their old (.html) paths to new paths
- Converting most Dactyl-specific syntax to Redocly equivalents
- Generating Redocly sidebar and redirects YAML files

This script is meant to be run from the repo top. It replaces syntax
in-place. Unless this is the final migration phase, the results of
running the migration script should be committed in a separate commit
whose message starts with '[DROP]' so it can be re-run on the latest
version of the master branch during rebasing.

Many commits have been squashed into this one, including:

- Add tool/migrate_links.sh as a one-stop conversion script for links.
- Enable the update_links filter in dactyl config but make it inactive
  unless you pass the appropriate vars
- Hack include_svg script to assume content/img instead of img

[FOLD] Migration scripting improvements:

- Roll scripting into all-in-one tool/migrate.sh
- Script moving/renaming Japanese snippets into @i18n
- Link replacment in snippets
- Handle links with query params
- Handle ref-links with anchors
- Remove some macro syntax that breaks Redocly
- Follow internal redirects in link replacement
- Handle links to some non-md pages

[FOLD] Migration script: handle more reflinks & imgs

[FOLD] tweak link migration

[FOLD] Fix substitution of reflinks

Add sidebar script

[FOLD] Fix link migration and whitespace noisiness

[FOLD] Link migration: auto-generate better link replacements

[FOLD] Convert badge syntax

[FOLD] Migration script: handle :not_enabled: syntax

[FOLD] Script generation of redirects

[FOLD] Migration script: make reusable common links

[FOLD] Fix common links code & conversion script comments

[FOLD] Add more non-md links

[FOLD] Fix filter_update_links syntax

[FOLD] Fix script's common links include placement

[FOLD] Migration script: update badge replacement to work w/ common-links

[FOLD] Fix ordering of converting common-links vs partials

[FOLD] Fix link substitution in common-links and fix trailing /index in redirects

mDuo13 committed:
Disable Dactyl CI on redocly branch

mDuo13 committed:
Commit package-lock to repo

mDuo13 committed:
Fix Redocly build

mDuo13 committed:
Restore some tsx pages to sidebars

mDuo13 committed:
Move about/resources/community pages appropriately

ckniffen committed:
fix: font-awesome font loading take 2

Some of the changes in #2295 are missing

ckniffen committed:
Migrate WebSocket Tool to Redocly

Recreate branch from base, add react-query-params, fix permalinks, fix sidebar

use correct params library and upgrade redocly.

Fix command text not working with permalink and move more modal logic out of main component.

Moved more connection selection logic to connection modal
Removed many `data-*` attributes previously used by bootstrap modal css

Created a shared modal component which removed 38 lines.

WS Tool: Fix Link import

fix UL error

toggle CurlModal to show/hide on button clicks

resolve error: <div> cannot appear as a descendant of <p>

remove <span>

WS tool: sidebar fixes

ckniffen committed:
feat: migrate rpc-tool to redocly

Differences between original:
- react18-json-view used for responses
- cleaned up markup for expand and collapse

Original work on this was done by @khancode

Co-authored-by: Omar Khan <khancodegt@gmail.com>

Fix RPC sidebar and width

mDuo13 committed:
Fix FAQ location issues

JST5000 committed:
Update domain-verifier

Add manifest outputs and cleaned up log

fix graceful handling of manifest errors and react key attribute

Update Domain Verifier sidebar entry

mDuo13 committed:
Migrate interactive tutorials (post realm v0.70)

Attempt migrating interactive tutorial again

Migrate interactive tutorial snippet syntax

Interactive tutorials: partially migrate send-xrp, no-freeze to new syntax

Fix Send XRP interactive tutorial

Interactive tutorials: fixes for Redocly incl. localization challenges

Interactive tutorials: switch defaultValue back to value in anticipation of Redocly bugfix

Fix document.ready→window.onRouteChange, cyclers, etc. in interactive
tutorials.

mDuo13 committed:
Bump Redocly to v0.69.3

mDuo13 committed:
Add working ignores

mDuo13 committed:
Improve code samples ignore

mDuo13 committed:
Bump Redocly to v0.69.4

AKCodez committed:
Update Redocly versions of Events/Community/Use Cases

updates events page to latest v

add keys to maps

community work

completes contribute/communnity page

uses page

use cases updates

Update content/community/events.page.tsx

Co-authored-by: Rome Reginelli <rome@ripple.com>

Update content/community/events.page.tsx

Co-authored-by: Rome Reginelli <rome@ripple.com>

Update content/community/index.page.tsx

Co-authored-by: Rome Reginelli <rome@ripple.com>

Update content/community/index.page.tsx

Co-authored-by: Rome Reginelli <rome@ripple.com>

q changes

fix errors

Update content/community/index.page.tsx

Co-authored-by: Rome Reginelli <rome@ripple.com>

package lock revert

Update content/community/index.page.tsx

Co-authored-by: Rome Reginelli <rome@ripple.com>

Update content/community/index.page.tsx

Co-authored-by: Rome Reginelli <rome@ripple.com>

Update content/community/index.page.tsx

Co-authored-by: Rome Reginelli <rome@ripple.com>

Update content/community/index.page.tsx

Co-authored-by: Rome Reginelli <rome@ripple.com>

Update content/community/index.page.tsx

Co-authored-by: Rome Reginelli <rome@ripple.com>

qa changes

moves uses into about

fix dupe key errors rm console log

fix wrong import

mDuo13 committed:
Bump Realm to 0.70.0

mDuo13 committed:
Fix crash on undefined child-pages

mDuo13 committed:
Move XRPLoader component to better place

mDuo13 committed:
Fix redocly/portal→realm import paths

mDuo13 committed:
Remove incompatible sidebar frontmatter

mahnunchik committed:
Add Coin Space Wallet

mDuo13 committed:
Bump realm to 0.71.1

mDuo13 committed:
Remove more redocly-incompatible syntax

mDuo13 committed:
Disable link subs for better link substitution

mDuo13 committed:
Remove redocly-incompatible syntax from checks tutorials

mDuo13 committed:
Bump Redocly to realm 0.74.1

mDuo13 committed:
Fix Redocly ignores stanza

mDuo13 committed:
Markdoc: add badge component

[FOLD] Add badge class to badges

mDuo13 committed:
Add not-enabled component

mDuo13 committed:
Deconditionalize common links for conversion

mDuo13 committed:
Bump Redocly to realm 0.74.2

mDuo13 committed:
Disable sidebars on About + Community pages

mDuo13 committed:
Add typescript logo to code-samples page

mDuo13 committed:
Remove inline partial from ledger_data

mDuo13 committed:
Remove obsolete validation_seed method page

mDuo13 committed:
Remove badges > ~2yrs old

Version 1.8.0 of rippled was released in late 2021, so any changes prior
to that are historical and no longer need badges.

Fixes #1978

mDuo13 committed:
Remove py-quickstart-intro.md (duplicate of send-payments-using-python.md)

mDuo13 committed:
Add tutorials landing & components

amarantha-k committed:
Migrate dev-blog to Redocly (#2346)

* Step 1 to migrate the blog - Add blog pages from the dev-blog repo

* Add blog-specific sidebar (& update contents)

* Add new dev reflections blog post from 01-23-2024

* Blog migration: fix markdoc errors and broken links

* Remove community pages from the sidebar

---------

Co-authored-by: mDuo13 <mduo13@gmail.com>

mDuo13 committed:
Add footer and frontmatter for custom pages (#2347)

Fix #2330

mDuo13 committed:
Update contributor documentation for Realm

mDuo13 committed:
Mass-replace blurb with seo:description in frontmatter

mDuo13 committed:
Add dev tools' frontmatter

mDuo13 committed:
Fix duplicate description frontmatter error

amarantha-k committed:
Add index file for Getting Started with PHP

mDuo13 committed:
Add frontmatter to homepage

mDuo13 committed:
Bump Redocly to Realm v0.74.3

mDuo13 committed:
Fix typo in ledger_data

mDuo13 committed:
Fix some interactive tutorial bugs

mDuo13 committed:
Fix top nav in Redocly

mDuo13 committed:
Fix top nav mobile styles

amarantha-k committed:
WIP Initial card-based reference landing page

amarantha-k committed:
Remove deprecated pages like Data API from the sidebar and update the metadata on these files to show the sidebar

amarantha-k committed:
Comment the GitHub code download syntax for now.

mDuo13 committed:
Top nav: Remove unused field, add note

mDuo13 committed:
Fix CSS styles

- Move docs graphics to appropriate place
- Fix spacing of header anchors
- Fix display of card grids in md
- Fix display of badges
- Fix theme-aware diagram coloring
- Fix left table columns being line-broken too aggressively

mDuo13 committed:
Remove now-unused _tutorials.scss

mDuo13 committed:
Fix code samples

mDuo13 committed:
CSS: Fix colors, footer styling

mDuo13 committed:
Fix broken links in partials and tutorials index

- Partials using common links need to be included as raw-partials
- xrpl-card href values need to use the production path, not the MD file
  path.

mDuo13 committed:
Fix broken [json method][] links

amarantha-k committed:
Updated hrefs on cards

amarantha-k committed:
Replace cards with a list for XRP-ledger.toml file

amarantha-k committed:
Fix image syntax on Stablecoin settings page

amarantha-k committed:
Update References landing page to have only 3 client libraries and add a link to view all

mDuo13 committed:
Fix docs page links

amarantha-k committed:
Fix syntax for image

amarantha-k committed:
Add new blog posted on 01-30-2024 to the redocly-migration branch

mDuo13 committed:
Bump Redocly to Realm v0.74.4

mDuo13 committed:
More Redocly style & key warning fixes:

- Fix unique key warnings on Dev Tools and Docs pages
- Fix active code tab colors in light mode
- Fix dev tools padding
- Fix dev tools cards in light mode
- Fix tx sender sidebar in light mode
- Fix domain verifier example code in light mode

mDuo13 committed:
Migrate content syntax via script

The changes in this commit were auto-generated by running

tool/migrate.sh

Following this commit, the Dactyl build no longer works but the Redocly
build (mostly) should.

mDuo13 committed:
Re-level non-docs content to top of repo and rename content→docs

mDuo13 committed:
Fix tsconfig paths on case-sensitive fs

mDuo13 committed:
Re-levelization: move non-docs content, rename content→docs

For better URLs, the content folder has been renamed 'docs' and all
other files have been moved up a level. Also, non-docs images have been
moved to the static folder at the top level where they belong.

Many relative paths had to be fixed to make this work.

mDuo13 committed:
Fix some key errors and remove some now unused files

mDuo13 committed:
Add blog post redirects

mDuo13 committed:
Reorg tutorials to match nav, and update links

mDuo13 committed:
Add a couple more redirects from old URLs

maria-robobug committed:
Update blog site for redocly migration

amarantha-k committed:
Resolve merge conflict caused by PR 2364

maria-robobug committed:
Fixed CSS not being loaded

amarantha-k committed:
Porting fix in PR#2368 to the redocly-migration branch

amarantha-k committed:
Bump Redocly to Realm v0.74.5

amarantha-k committed:
Remove extraneous HTML code in title

amarantha-k committed:
Merge pull request #2369 from XRPLF/migration-fixes-0202

Fixes to issues found when testing the Redocly-based preview build

amarantha-k committed:
Update to display syntax as-is

AKCodez committed:
adds eth denver

amarantha-k committed:
Hide submit code sample button which is not linked and the button is not on the live site currently

AKCodez committed:
add cypress tech odyssey event

AKCodez committed:
adds remaining events

AKCodez committed:
Update community/index.page.tsx

Co-authored-by: Amarantha Kulkarni <amarantha-k@users.noreply.github.com>

AKCodez committed:
Merge pull request #2371 from XRPLF/events-updates-2024-02-02

Events updates 2024 02 02

amarantha-k committed:
Fix issue where some wallet icons are not displaying properly

amarantha-k committed:
Fix image url

amarantha-k committed:
Reorder events so upcoming events are displayed chronologically

JST5000 committed:
Update xrpl usage within xrpl-dev-portal

JST5000 committed:
Update build-a-browser-wallet

JST5000 committed:
Update build-a-desktop-wallet

JST5000 committed:
Update checks snippets

JST5000 committed:
Fix create-amm js

JST5000 committed:
Migrate create-amm ts

JST5000 committed:
Update escrow code snippet

JST5000 committed:
Update freeze codesamples

JST5000 committed:
Update get-started js snippet

JST5000 committed:
Update issue-a-token

maria-robobug committed:
Fetch blog post data from md file frontmatter

amarantha-k committed:
Merge pull request #2373 from XRPLF/migration-testing-0202

Fixes from migration testing

AKCodez committed:
fix soft wallet className issue

AKCodez committed:
fix icons in light / dark mode

AKCodez committed:
Merge pull request #2372 from XRPLF/add-cswallet-redocly

Add cswallet redocly

amarantha-k committed:
Porting updates from  PR #2271 to the Redocly migration branch

amarantha-k committed:
Porting updates from PR #2193 to the redocly-migration branch

AKCodez committed:
added logic utilizing existing bootstrap functionality to close sidebar after dropdown-item click on mobile

amarantha-k committed:
Merge pull request #2376 from XRPLF/mobile-navbar-fix

added logic utilizing existing bootstrap functionality to close sideb…

JST5000 committed:
Update key-derivation sample

JST5000 committed:
Update require-destination-tags

JST5000 committed:
Update send-a-memo

JST5000 committed:
Update send-xrp

JST5000 committed:
Update trade-in-the-exchange

JST5000 committed:
Update tx-serialization

JST5000 committed:
Update use-tickets

JST5000 committed:
Update build-a-desktop-wallet deps

maria-robobug committed:
Landing page CSS

amarantha-k committed:
Merge pull request #2378 from XRPLF/3.0-redocly

Update xrpl.js to 3.0 across xrpl.org

amarantha-k committed:
Merge pull request #2377 from XRPLF/migrate-open-prs-0205

Port updates from open PRs to the redocly-migration branch

amarantha-k committed:
Rename file to index.md to fix duplicate names in the URL

amarantha-k committed:
Fix plaintext color in light mode

AKCodez committed:
adds seo title description and h1 changes

amarantha-k committed:
Fix idify function in interactive-tutorials.js to produce same results as the markdoc tag

amarantha-k committed:
Update links after renaming accounts.md to index.md

AKCodez committed:
Update automated-market-makers.md

Co-authored-by: Amarantha Kulkarni <amarantha-k@users.noreply.github.com>

AKCodez committed:
Update index.page.tsx

Co-authored-by: Amarantha Kulkarni <amarantha-k@users.noreply.github.com>

AKCodez committed:
Update docs/concepts/payment-types/index.md

Co-authored-by: Amarantha Kulkarni <amarantha-k@users.noreply.github.com>

AKCodez committed:
Update index.page.tsx

Co-authored-by: Amarantha Kulkarni <amarantha-k@users.noreply.github.com>

AKCodez committed:
Update index.page.tsx

Co-authored-by: Amarantha Kulkarni <amarantha-k@users.noreply.github.com>

AKCodez committed:
Update index.page.tsx

Co-authored-by: Amarantha Kulkarni <amarantha-k@users.noreply.github.com>

amarantha-k committed:
Merge pull request #2382 from XRPLF/xrpl-redocly-seo

adds seo title description and h1 changes

amarantha-k committed:
Add a missing space

amarantha-k committed:
Comment out Submit Code Samples link

amarantha-k committed:
Update footer text

amarantha-k committed:
Fix links on Docs home page

amarantha-k committed:
Fix links on tutorials home page

amarantha-k committed:
Fix the numbering on XRP Exchanges

amarantha-k committed:
Remove extraneous <br> tag

amarantha-k committed:
Merge pull request #2383 from XRPLF/migration-testing-0206

Migration testing 0206

amarantha-k committed:
Bump Redocly Realm to 0.75.0

amarantha-k committed:
Add missing closing parenthesis

amarantha-k committed:
Update idify in helper.ts

amarantha-k committed:
Fix source file path

amarantha-k committed:
Merge pull request #2387 from XRPLF/bump-redocly-0.75

Bump Redocly Realm to 0.75.0

maria-robobug committed:
Resolve generated CSS conflict

maria-robobug committed:
Update Navigation for Blog site

amarantha-k committed:
Update clawback status to enabled

amarantha-k committed:
Fix links to getting started with SDK tutorials

DennisDawson committed:
Update index.page.tsx

Fix links to Javascript, Python, and Java pages under Explore SDKs

RomanHotsiy committed:
Fix interactive tutorials issues

RomanHotsiy committed:
Fix interactive tutorials issues

amarantha-k committed:
Merge pull request #2391 from RomanHotsiy/fix-tutorials

Fix interactive tutorials

amarantha-k committed:
Fix typo - missing double quotation mark

amarantha-k committed:
Merge pull request #2390 from XRPLF/migration-fixes-0208

Porting updates (clawback enablement) and fixing bugs found during testing.

maria-robobug committed:
Add light mode CSS

maria-robobug committed:
Add mobile dropdown

maria-robobug committed:
Merge pull request #2388 from maria-robobug/blog-site

Blog site

amarantha-k committed:
Resolved merge conflicts
Merge branch 'master' into redocly-migration

AKCodez committed:
fixes a tag issues

AKCodez committed:
Merge pull request #2392 from XRPLF/issue-2384

fixes a tag issues

amarantha-k committed:
Update config to generate sitemap

amarantha-k committed:
Merge pull request #2393 from XRPLF/testing-fixes-0213

Update config to generate sitemap

amarantha-k committed:
Add rbac config for xrpl-org-editors team

amarantha-k committed:
Add missing space in heading

amarantha-k committed:
Merge pull request #2395 from XRPLF/rbac-xrpl-project

Add rbac config for xrpl-org-editors team

maria-robobug committed:
Add Clio 2.1.0 release notes

AKCodez committed:
fix conflicting color

maria-robobug committed:
Merge pull request #2396 from maria-robobug/clio-release-notes

Add Clio 2.1.0 release notes

maria-robobug committed:
Add Clio release notes to blog sidebar navigation.

Small change: I missed adding the latest Clio release notes file to the blog sidebar navigation.
Plus some minor formatting fixes.

oeggert committed:
initial draft

oeggert committed:
initial draft

oeggert committed:
typo fix

oeggert committed:
typo fix

oeggert committed:
2.1.0 amendment updates

oeggert committed:
2.1.0 amendment updates

oeggert committed:
Merge pull request #2403 from XRPLF/2.1.0-updates

2.1.0 amendment updates

amarantha-k committed:
Update color definition to fix search window in light mode

amarantha-k committed:
Revert defn for color-gray-10

amarantha-k committed:
Merge pull request #2405 from XRPLF/issue-2385-3

Update color definition to fix search window in light mode

amarantha-k committed:
Revert change to base color and add bg-base for light mode

justinr1234 committed:
feat: Osano for Redocly

amarantha-k committed:
Merge pull request #2406 from XRPLF/issue-2385-4

Revert change to base color and add bg-base for light mode

amarantha-k committed:
Merge pull request #2397 from XRPLF/issue-2385-2

fix conflicting color

amarantha-k committed:
Merge pull request #2400 from maria-robobug/redocly-migration

Add Clio release notes to blog sidebar navigation.

justinr1234 committed:
fix: osano styling box-sizing

justinr1234 committed:
feat: add xrpl ai

oeggert committed:
rebas from master

oeggert committed:
update dactyl

amarantha-k committed:
Merge pull request #2407 from XRPLF/osano-redocly

feat: Osano & XRPL AI for Redocly

amarantha-k committed:
Merge pull request #2410 from XRPLF/port-release-info

2.1.0 amendment updates

amarantha-k committed:
Merge branch 'master' into redocly-migration

tequdev committed:
[JA] update known-amendment (#2410)

tequdev committed:
move @i18n/ja/concepts to @i18n/ja/docs/concept

tequdev commented on dactyl-config.yml:
```diff
@@ -680,6 +680,10 @@ pages:
             - ja
 
+    -   md: use-cases/defi/integrate-with-xrpl-dex.md
+        targets:
+            - en
```
```suggestion
        targets:
            - en
            - ja
```

amarantha-k committed:
Add translations.yaml file - try migrating key-value pairs of already translated content

amarantha-k committed:
Add more key-value pairs of translated text

amarantha-k committed:
Remove extra space

amarantha-k committed:
Fix special characters

amarantha-k committed:
Add translation keys for nav bar and sidebar items

amarantha-k committed:
Merge pull request #2414 from tequdev/ja-move-concepts

move `@i18n/ja/concepts` to `@i18n/ja/docs/concept`

amarantha-k committed:
Merge pull request #2412 from tequdev/ja-known-amendment-20240226

[JA] Update known amendment

amarantha-k committed:
Merge pull request #2415 from XRPLF/i18n-migrate-po-file

Migrate key-value pairs from message.po file to translations.yaml file

amarantha-k committed:
WIP readme update for Redocly

amarantha-k committed:
Merge pull request #2336 from XRPLF/redocly-migration

Migrate Toolchain to Redocly

amarantha-k committed:
Update readme to reflect toolchain update from Dactyl to Redocly

amarantha-k committed:
Incorporate review feedback

amarantha-k committed:
Merge pull request #2420 from XRPLF/readme-update

Readme update

amarantha-k committed:
Add GA tracking ID

AKCodez committed:
fix links on code samples page

amarantha-k committed:
Update field to ga instead of gtm

amarantha-k committed:
Merge pull request #2422 from XRPLF/card-links

fix links on code samples page

amarantha-k committed:
Merge pull request #2421 from XRPLF/add-ga

Add GA tracking ID

tequdev committed:
[JA] update nftokenacceptoffer

tequdev committed:
[JA] fix snippets path

maria-robobug committed:
AMM transaction types in account_tx (Clio)

Now that Clio 2.1.0 has been released, we want to make sure this info is updated in the api methods.

Ports changes made in this draft [PR](https://github.com/XRPLF/xrpl-dev-portal/pull/2337).

I will close that draft once this is merged.

amarantha-k committed:
Fix links to docs pages

amarantha-k committed:
Merge pull request #2423 from tequdev/ja-nftokenacceptoffer

[JA] update NFTokenAcceptOffer page

amarantha-k committed:
Merge pull request #2424 from tequdev/ja-snippets

[JA] fix snippets paths

amarantha-k committed:
Merge pull request #2426 from maria-robobug/clio-account-tx-amm

AMM transaction types in account_tx (Clio)

amarantha-k committed:
add translations key to navbar.tsx

amarantha-k committed:
Add translation keys for footer headings and sidebar items

amarantha-k committed:
Merge pull request #2427 from XRPLF/fixes-28feb24

oeggert committed:
migrate 2.1 release notes

amarantha-k committed:
Merge pull request #2430 from XRPLF/fix-2.1-rn

migrate 2.1 release notes

oeggert committed:
Merge branch 'dex-integration' of https://github.com/XRPLF/xrpl-dev-portal into dex-integration

oeggert commented:
Closing this PR. Refer to #2433
